app/ide-desktop/lib/dashboard/src/tailwind.css

```diff
@@ -217,7 +217,7 @@
   --side-panel-label-width: 8rem;
   --side-panel-description-padding-y: 0.25rem;
 
-  --extended-editor-menu-size: 2.5rem;
+  --extended-editor-menu-size: 48px;
```

Comment:
for reference, 1rem = 16px. but i suppose this *is* more robust as the menu will always be a fixed pixel size.